apps/laboratory/src/pages/library/wagmi-email.tsx

```diff
@@ -14,6 +13,7 @@ export const wagmiConfig = defaultWagmiConfig({
   chains: WagmiConstantsUtil.chains,
   projectId: ConstantsUtil.ProjectId,
   metadata: ConstantsUtil.Metadata,
+  ssr: true,
```

Comment:
Not a big fan of this, can we instead check internally for something like `window === 'undefined'` instead and pass that on to wagmi?

Reply:
I'm not against it, but maybe some user needs to disable this for some reason and wouldn't be able to. 
Maybe the flag can be on by default, and we check for `window` on our side as an extra check and they can turn it off completely

Reply:
I think this is great for the examples since this is handled by Wagmi already, we just need to add the `cookies` feature imo